site-src/v1alpha2/guides/getting-started.md

```diff
@@ -1,10 +1,7 @@
 # Getting started with Gateway APIs
 
-!!! danger
-    This page has not been updated for v1alpha2 yet.
-
```

Comment:
Right, because this is the landing page so this should be the last one to remove its banner, makes sense to me in retrospect.

Given that part of the context here seems to be that it's the pages this page _links to_ that are not updated yet, I added this back in b6a39a62adfacb0269602c965231ecdcd2e2e56c but I changed the wording a bit, lmkwyt :+1: